docs/src/codegen.md

```diff
@@ -156,55 +192,121 @@ playwright codegen --color-scheme=dark playwright.dev
 
 ```bash js
-npx playwright codegen --save-storage=auth.json
+npx playwright codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
 ```bash java
-mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="codegen  --save-storage=auth.json"
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="codegen github.com/microsoft/playwright  --save-storage=auth.json"
 ```
 
 ```bash python
-playwright codegen --save-storage=auth.json
+playwright codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
 ```bash csharp
-pwsh bin/Debug/netX/playwright.ps1 codegen --save-storage=auth.json
+pwsh bin/Debug/netX/playwright.ps1 codegen github.com/microsoft/playwright --save-storage=auth.json
 ```
 
-<img width="1264" alt="Screenshot 2022-08-03 at 13 28 02" src="https://user-images.githubusercontent.com/13063165/182599605-df2fbd05-622b-4cd7-8a32-0abdfea7d38d.png" />
+######
+* langs: js
+
+<img width="1394" alt="github page before logging in js" src="https://user-images.githubusercontent.com/13063165/220558833-67ef96a9-7278-4698-b04a-f6a20e93cbe1.png" />
+
+######
+* langs: java
+
+<img width="1394" alt="github page before logging in java" src="https://user-images.githubusercontent.com/13063165/220559095-94f127d7-d8d9-4c56-ba58-5f62e91dfe84.png" />
+
+
+######
+* langs: python
+
+<img width="1394" alt="github page before logging in python" src="https://user-images.githubusercontent.com/13063165/220559267-8c6ca9df-0c69-4b6c-8a1c-e5d022a1f6bc.png" />
+
+######
+* langs: csharp
+
+<img width="1394" alt="github page before logging in csharp" src="https://user-images.githubusercontent.com/13063165/220559407-6f124c3a-ac9d-4006-bf03-ae58917f7695.png" />
 
-Run with `--load-storage` to consume previously loaded storage. This way, all [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) and [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) will be restored, bringing most web apps to the authenticated state without the need to login again.
+#### Login
+
+After performing authentication and closing the browser, `auth.json` will contain the storage state which you can then reuse in your tests. 
+
+<img width="1394" alt="login to Github screen" src="https://user-images.githubusercontent.com/13063165/220561688-04b2b984-4ba6-4446-8b0a-8058876e2a02.png" />
+
+Make sure you only use the `auth.json` locally as it contains sensative information. Add it to your `.gitignore` or delete it once you have finished generating your tests.
+
+<img width="1385" alt="vscode showing auth.json" src="https://user-images.githubusercontent.com/13063165/220561822-d8e190c8-0064-4c53-a87a-0165d2c60fce.png" />
```

Comment:
I think the content of it is not relevant, and its an implementation detail. Not worth showing an image.